packages/@cdktf/hcl2cdk/lib/expressions.ts

```diff
@@ -130,6 +129,30 @@ export async function extractReferencesFromExpression(
   }, [] as Reference[]);
 }
 
+function getResourceNamespace(
+  provider: string,
+  resource: string,
+  isDataSource: boolean
+) {
+  if (provider === "cdktf") {
```

Comment:
When does this case happen?

Reply:
When we have `cdktf.TerraformStack` for example